### ISSUE TITLE
sqlite3 move from base to ports

### ISSUE THREAD

**Issue by mattieub** [torbsd/openbsd-ports#7]
Hello,
The port doesn't compile anymore since the move of sqlite3 from base to ports.
I tried to make the same change than firefox-esr, but the build fails in security/nss, which doesn't pick "-I${LOCALBASE}/include".
Have you an idea on what's going on?
Cheers.

**Comment by haqistan** [comment]
Yes, I'm aware of the issue. I investigated a quick fix but I don't think it's the way to go. I also hacked my Makefile in several different ways to try and get -I${LOCALBASE}/include passed into the nss build, will look at it fresh today. My hack (in the develop branch if you want to look) was to switch to the bundled sqlite, like www/mozilla-firefox uses, just to see if it built. It does, but I don't think I have the options right and I'm not sure this is a good thing anyway.

Just FYI we aren't using security/nss, we're using the built-in nss since a couple versions because that's what TBB does - they have hacked nss, we can't use security/nss anymore.  The nss build system and its relationship with the moz build system is ... complex.

In any event thanks for reporting this. I'm on it but please chime in here if you figure anything out, or come get me on IRC if you prefer. Pax, -A

**Comment by haqistan** [comment]
Okay I've looked into this a bit now. It appears building with the bundled sqlite3 is in fact the right thing to do. I have to fix a couple things and double-check that the build is still good but it looks like this is the way forward.
